physics/Magnetized Cold Plasma Properties.ipynb

```diff
@@ -0,0 +1,272 @@
+      "text/plain": [
+       "<Figure size 864x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "plt.semilogx(f, abs(S_pos),\n",
+    "             f, abs(D_pos),\n",
+    "             f, abs(P_pos), lw=2)\n",
+    "plt.semilogx(f, abs(S_neg), '#1f77b4',\n",
+    "             f, abs(D_neg), '#ff7f0e',\n",
+    "             f, abs(P_neg), '#2ca02c', lw=2, ls='--')\n",
```

Comment:
```suggestion
    "plt.semilogx(f, abs(S_neg), 'C1',\n",
    "             f, abs(D_neg), 'C2',\n",
    "             f, abs(P_neg), 'C3', lw=2, ls='--')\n",
```

I think this is going to do what you want it to do (get the colors from the default color map), though I'm not sure the suggestion interface really works well for code suggestions in notebooks. Might be better to do this manually.

Reply:
Ho, nice trick. I didn't know that one. Thanks.

It should be 'C0', 'C1', 'C2' however.

Reply:
That it should indeed, and is what I did in the commit I sent in :)